src/iconsManifest/supportedExtensions.ts

```diff
@@ -576,32 +576,18 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'babel',
-      extensions: [
-        '.babelrc',
-        'babelrc.js',
-        '.babelrc.js',
-        'babelrc.json',
-        'babel.config.js',
-        'babel.config.esm.js',
-        'babel.config.json',
-        '.babelignore',
-      ],
+      extensions: ['.babelrc', '.babelignore', 'babel.config.esm.js'],
```

Comment:
This wrong. Add `esm.js` in `extensionsGlob`.

Reply:
Oh wait. It will be tide to `babelrc` as well.

Reply:
@JimiC We don't know if `.babelrc.esm.js` exists unless we can find the original docs or source codes for it.

Reply:
Yeah saw it afterwards. Let's leave as it is.

Reply:
I think that `.babelrc.esm.js` not exist (`.babelrc` is for static config). `gulp.config.esm.js` exists for autodetect & load ESM syntax Javascript files on Gulp through Babel.

Reply:
@ManzDev we need info about `babel.config. esm.js`°

Reply:
I believe that `babel.config.esm.js` can be deleted. I needed it to make an independent configuration of CommonJS, but it could be done from one. It worked for me at the time, but I don't see any official reference either.